src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started transporting materials.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped transporting materials.");

```

Comment:
Similarly, the stop message should reflect that it is a Truck. For instance: 'Truck is working: stopped transporting materials.'